src/Symfony/Component/Validator/ValidatorBuilder.php

```diff
@@ -38,6 +39,8 @@
  * The default implementation of {@link ValidatorBuilderInterface}.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @final since Symfony 4.4
```

Comment:
We missed the fact that the interface is deprecated. We cannot do this anymore.

Reply:
😢